packages/gestalt/src/Popover.js

```diff
@@ -1,24 +1,60 @@
 // @flow strict
 import type { Node } from 'react';
 import Controller from './Controller.js';
-import { type Role } from './Contents.js';
 
 type Color = 'blue' | 'orange' | 'red' | 'white' | 'darkGray';
 type Size = 'xs' | 'sm' | 'md' | 'lg' | 'xl' | 'flexible' | number;
 type IdealDirection = 'up' | 'right' | 'down' | 'left';
+type Role = 'dialog' | 'listbox' | 'menu';
 
 type Props = {|
+  /**
+   * The reference element, typically [Button](https://gestalt.pinterest.systems/button) or [IconButton](https://gestalt.pinterest.systems/iconbutton), that Popover uses to set its position
+   */
   anchor: ?HTMLElement,
+  /**
+   * The content shown in Popover
+   */
   children?: Node,
+  /**
+   * The background color of Popover. See the [color and caret](https://gestalt.pinterest.systems/popover#Color-and-caret) variant to learn more.
+   */
   color?: Color,
+  /**
+   *
+   */
   handleKeyDown?: (event: SyntheticKeyboardEvent<HTMLElement>) => void,
```

Comment:
Huh, I never noticed this doesn't conform to our event standard. We should probably update that at some point

Reply:
I think it was added as an internal prop, but agree, we should refactor it

Reply:
I _think_ we're not using this anywhere in Pinboard, so perhaps we should add a `_` prefix to make it officially internal?